integration/nwo/fabric/fpc/fpc.go

```diff
@@ -112,15 +112,11 @@ func (n *Extension) GenerateArtifacts() {
 	}
 }
 
-func (n *Extension) PostRun(load bool) {
+func (n *Extension) PostRun(bool) {
```

Comment:
does static check complain if we keep the variable but not using it? Ideally, we can keep the method header unchanged

Reply:
It doesn't complain indeed. I removed the variable, because it's marked as unused in the IDE and the other implementations of the `PostRun` method don't define the variable. Let me revert it back